packages/client/src/game/utils/isOdd.ts

```diff
@@ -0,0 +1,7 @@
+export const isOdd = (number: number): boolean => {
+  if (number % 2) {
```

Comment:
можно заменить на всю конструкцию на

```suggestion
  return number % 2;
```